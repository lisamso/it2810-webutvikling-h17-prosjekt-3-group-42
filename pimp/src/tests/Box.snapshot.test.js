import Box from '../components/Box';
import React from 'react';
import { shallow, mount, render } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

test('Box component should render as expected', () =>{
  const component = shallow(<Box />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
