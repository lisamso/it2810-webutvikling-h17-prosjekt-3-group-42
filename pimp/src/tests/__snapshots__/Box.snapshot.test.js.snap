// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box component should render as expected 1`] = `
<div
  className="box undefined"
>
  <h2
    id="box-title"
  />
  <hr />
  <div
    className="box-container"
  />
  <div
    id="btn"
  />
</div>
`;
